blog/authors.yml

```diff
@@ -39,3 +39,9 @@ xander-vedder:
   title: Stryker Team
   url: https://github.com/xandervedder
   image_url: https://avatars.githubusercontent.com/u/34238405?v=4
+
+rachid-boukaiba:
+  name: Rachid Boukaiba
+  title: Stryker.NET Team
+  url: https://rach.id
```

Comment:
The url doesn't really show anything. Maybe it's better to link to your github page? Unless you plan to add anything to your site ;)